.ddev/commands/web/install

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+## Description: Run the az_quickstart installation inside the web container.
+## Usage: install
+## Example: "ddev install"
+
+rm -rf /var/www/html/docroot && rm /var/www/html/index.html
+git clone https://github.com/az-digital/az-quickstart-scaffolding.git /var/www/html
```

Comment:
Do we want to add the step that checks if a branch of the same name exists for the scaffolding repo here (that we use in or Lando and ProboCI configs)?

See: https://github.com/az-digital/az_quickstart/blob/master/.lando.yml#L22 and https://github.com/az-digital/az_quickstart/blob/master/.probo.yaml#L10

Reply:
I added in the same line from the lando file. I also added in the other tooling that's in that file, namely phpcs, drupal, and drupal-check.

Reply:
Looks good.  Thanks!